<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>882</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>852</width>
    <height>882</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>852</width>
    <height>882</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cascadia Code PL</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MiniBasic - Made by JolyneFr</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/b.svg</normaloff>:/b.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>811</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cr_layout">
     <item>
      <layout class="QVBoxLayout" name="code_layout">
       <item>
        <widget class="QLabel" name="code_label">
         <property name="font">
          <font>
           <family>Cascadia Code PL</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="code_display"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="result_layout">
       <item>
        <widget class="QLabel" name="result_label">
         <property name="font">
          <font>
           <family>Cascadia Code PL</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="result_display"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>811</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ss_layout">
     <item>
      <widget class="QLabel" name="ss_label">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Statement and Syntax Tree </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="ss_display"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>811</width>
      <height>75</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>In fact, I recommend you edit your program in notebook first.</string>
       </property>
       <property name="text">
        <string> Load and Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>In fact, I recommend you edit your program in notebook first.</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>In fact, I recommend you edit your program in notebook first.</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="del_Layout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>from:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="del_from"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>to:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="del_to"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="delete_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Cascadia Code PL</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="statusTip">
          <string>In fact, I recommend you edit your program in notebook first.</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>811</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ciw_layout">
     <item>
      <widget class="QLabel" name="ciw_label">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Command Input Window </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ciw_input">
       <property name="font">
        <font>
         <family>Cascadia Code PL</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>790</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial&quot;;
border-color: rgb(170, 0, 0);
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>No Error Detected So Far.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>25</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>In fact, I recommend you edit your program in notebook first.</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="statusTip">
     <string>In fact, I recommend you edit your program in notebook first.</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_load"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="statusTip">
    <string>In fact, I recommend you edit your program in notebook first.</string>
   </property>
  </widget>
  <action name="action_save">
   <property name="text">
    <string>Save to File</string>
   </property>
   <property name="statusTip">
    <string>In fact, I recommend you edit your program in notebook first.</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>Load &amp; Run File</string>
   </property>
   <property name="statusTip">
    <string>In fact, I recommend you edit your program in notebook first.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
